python/mxnet/contrib/onnx/mx2onnx/_op_translations.py

```diff
@@ -1353,7 +1353,7 @@ def convert_min(node, **kwargs):
     mx_axis = node.get("attrs", {}).get("axis", None)
     axes = convert_string_to_list(str(mx_axis)) if mx_axis is not None else None
 
-    keepdims = int(node.get("attrs", {}).get("keepdims", 0))
+    keepdims = get_boolean_attribute_value(node.get("attrs", {}), "keepdims")
```

Comment:
same as above, and in all other instance in this PR.

Reply:
yea, correct. my comment was for places where you had a redundant if else statement like above. This is good the way it it.

Reply:
Hopefully this is the last try, then.